sbt-jib/src/main/scala/de/gccc/jib/JibPlugin.scala

```diff
@@ -102,19 +107,20 @@ object JibPlugin extends AutoPlugin {
     jibTags                        := List.empty,
     Jib / mappings                 := Nil,
     JibExtra / mappings            := Nil,
-    jibMappings                    := (Jib / mappings).value,
-    jibExtraMappings               := (JibExtra / mappings).value,
-    jibExtraMappingPermissions     := Nil,
+    jibMappings                    := PluginCompat.uncached((Jib / mappings).value),
```

Comment:
The inherited `mappings` task is explicitly uncached in sbt 2 and that has transitive effect on many tasks downstream. I'm not sure if there's a better way to go at it.

Without, you get this compilation error:
```
given evidence sjsonnew.JsonFormat[scala.collection.immutable.Seq[scala.Tuple2[de.gccc.jib.PluginCompat.FileRef, scala.Predef.String]]] is not found; opt out of caching by annotating the key with @cacheLevel(include = Array.empty), or asfoo := Def.uncached(...), or provide a given value
```